test/quantization/test_quantized_module.py

```diff
@@ -728,3 +728,48 @@ def test_linear_api(self, batch_size, in_features, out_features, use_bias, use_d
 
         # Smoke test extra_repr
         self.assertTrue('QuantizedLinear' in str(quantized_float_linear))
+
+    def test_lstm_api(self):
+        r"""Test execution and serialization for dynamic quantized lstm modules on int8 and fp16
+        """
+        # Check that module matches the numerics of the op and ensure that module can be
+        # instantiated for all engines and dtypes
+
+        for qengine in supported_qengines:
+            with override_quantized_engine(qengine):
+                for dtype in [torch.qint8, torch.float16]:
+                    if dtype == torch.float16 and qengine == "qnnpack":
+                        # fp16 dynamic quant is not supported for qnnpack
+                        continue
+                        # Test default instantiation
+                    seq_len = 128
+                    batch = 16
+                    input_size = 3
+                    hidden_size = 7
+                    num_layers = 2
+                    bias = True
+                    bidirectional = False
+
+                    x = torch.rand(seq_len, batch, input_size)
+                    h = torch.rand(num_layers * (bidirectional + 1), batch, hidden_size)
+                    c = torch.rand(num_layers * (bidirectional + 1), batch, hidden_size)
+
+
+                    cell_dq = torch.nn.quantized.dynamic.LSTM(input_size=input_size,
+                                                              hidden_size=hidden_size,
+                                                              num_layers=num_layers,
+                                                              bias=bias,
+                                                              batch_first=False,
+                                                              dropout=0.0,
+                                                              bidirectional=bidirectional,
+                                                              dtype=dtype)
+                    _all_params = ([m.param for m in cell_dq._all_weight_values])
+                    result = torch.quantized_lstm(x, (h, c), _all_params, cell_dq.bias, cell_dq.num_layers,
```

Comment:
Do we also have a test for the quantize_dynamic api for LSTM modules? Should we test that as well?